cookbooks/bcpc-hadoop/attributes/monitoring.rb

```diff
@@ -0,0 +1,121 @@
+         'enable' => false
+      }
+   },
+   'hbase_master' => {
+      'hbasenonheapmem' => {
+         'type' => "jmx",
+         'query' => "memory.NonHeapMemoryUsage_committed",
+         'trigger_val' => "max(61,0)",
+         'value_type' => 3,
+         'trigger_cond' => "=0",
+         'trigger_name' => "HBaseMasterAvailability",
+         'enable' => false,
+         'trigger_dep' => ["NameNodeAvailability"],
+         'trigger_desc' => "HBase master seems to be down",
+         'severity' => 5,
+         'route_to' => "admin"
+      },
+      'hbaseheapmem' => {
+         'type' => "jmx",
+         'query' => "memory.HeapMemoryUsage_committed",
+         'history_days' => 2,
+         'trend_days' => 30,
+         'enable' => false
+      },
+      'numrsservers' => {
+         'type' => "jmx",
+         'query' => "hbm_server.Master.numRegionServers",
+         'trigger_val' => "max(61,0)",
+         'value_type' => 3,
+         'trigger_cond' => "=0",
+         'trigger_name' => "HBaseRSAvailability",
+         'enable' => false,
+         'trigger_dep' => ["HBaseMasterAvailability"],
+         'trigger_desc' => "HBase region server seems to be down",
+         'severity' => 5,
+         'route_to' => "admin"
+      }
+   },
+   'bcpc-vm' => {
+      # Entry in suffix array added to hostname as suffix and 
+      # to key name, trigger name as suffix "_<hostname with suffix>"
+      # If specified, should be *first* entry for a host
+      'suffix' =>  [ "1", "2", "3" ], 
+      'memory_active' => {
+         'type' => "servers",
+         'query' => "memory.Active",
+         'trigger_val' => "max(61,0)",
+         'trigger_cond' => "=0",
+         'trigger_name' => "NodeAvailability",
+         'enable' => true,
+         'trigger_desc' => "Node seems to be down",
+         'severity' => 5,
+         'route_to' => "admin"
+      },
+      'disk_space_avail' => {
+         'type' => "servers",
+         'query' => "diskspace._disk_*.byte_avail",
+         'trigger_val' => "max(61,0)",
+         'value_type' => 3,
+         'trigger_cond' => "=0",
+         'trigger_name' => "DiskAvailability",
+         'enable' => false,
+         'trigger_desc' => "Disk seems to be full or down",
+         'severity' => 4,
+         'route_to' => "admin"
+      },
+      'disk_space_used' => {
+         'type' => "servers",
+         'query' => "diskspace._disk_*.byte_used",
+         'trigger_val' => "max(61,0)",
+         'value_type' => 3,
+         'trigger_cond' => ">15G",
+         'trigger_name' => "DiskSpaceUsage",
+         'enable' => false,
+         'trigger_desc' => "More than 15G disk used",
+         'severity' => 3,
+         'route_to' => "tenant"
+      },
+      'chef_client_time' => {
+         'type' => "chef",
+         'query' => "elapsed_time",
+         'trigger_val' => "max((#{node['chef_client']['interval']}.to_i + #{node['chef_client']['splay']}.to_i ) / 60 ,0)",
```

Comment:
The Zabbix [max](https://www.zabbix.com/documentation/2.2/manual/appendix/triggers/functions) function seems to take seconds, here you are dividing seconds (the Chef run interval) by 60?